cves/CVE-2011-3949.yml

```diff
@@ -268,8 +290,10 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: |
+    The vulnerability involved uploading custom built malicious files, which is
```

Comment:
If the vulnerability involved constructed files, make sure you specify that in your vulnerability description.